spec/masamune/filesystem_spec.rb

```diff
@@ -337,9 +337,11 @@
 
     context 'hdfs' do
       before do
+        expect(filesystem).to receive(:hadoop_fs).with('-mkdir', '-p', 'file://' + old_dir).once
+        expect(filesystem).to receive(:hadoop_fs).with('-touchz', 'file://' + new_file, 'file://' + other_new_file).once
         instance.touch!('file://' + new_file, 'file://' + other_new_file)
       end
-      it { is_expected.to eq(true) }
+      it 'meets expectations' do; end
```

Comment:
Minor style:  To avoid the not-very-meaningful empty it-meets-expectations block, consider either just using an it-block containing everything currently in your before block, or (alternatively) an allow-to-receive in a before block and then expect-to-have-received validations in the it-block after the action (https://relishapp.com/rspec/rspec-mocks/docs/basics/spies).